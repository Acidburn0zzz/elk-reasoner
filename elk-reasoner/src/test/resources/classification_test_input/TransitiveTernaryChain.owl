Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#transitive_ternary_chain>
TransitiveObjectProperty(:T)
SubObjectPropertyOf(ObjectPropertyChain(:T :T :T) :H)
SubClassOf(:A ObjectSomeValuesFrom(:T :B))
SubClassOf(:B ObjectSomeValuesFrom(:T :C))
SubClassOf(:C ObjectSomeValuesFrom(:T :D))
EquivalentClasses(:X ObjectSomeValuesFrom(:H :D))
)