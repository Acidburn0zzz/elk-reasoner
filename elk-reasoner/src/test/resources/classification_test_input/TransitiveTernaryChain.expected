Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#transitive_ternary_chain#taxonomy>
SubClassOf(:A :X)
Declaration(Class(:B))
Declaration(Class(:C))
Declaration(Class(:D))
)